src/Authenticator/CookieAuthenticator.php

```diff
@@ -73,7 +73,7 @@ public function __construct(IdentifierCollection $identifiers, array $config = [
      */
     protected function _checkCakeVersion()
     {
-        if (!class_exists('Cake\Http\Cookie\Cookie')) {
+        if (!class_exists(Cookie::class)) {
```

Comment:
This needs to be reverted otherwise it will generate an error when the class doesn't exist, which is exactly what this check is trying to avoid.

Reply:
@ADmad `::class` syntax never generates any errors. See a note of the following:
http://php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class

Reply:
@chinpei215 Thanks, good to know.